src/ios.rs

```diff
@@ -2,12 +2,14 @@ use std::{ffi::OsStr, io, process::Command};
 
 use crate::{CommandExt, IntoResult};
 
+pub fn command<T: AsRef<OsStr>>(path: T) -> Command {
```

Comment:
The signature here could be 

```suggestion
pub fn commands<T: AsRef<OsStr>>(path: T) -> impl Iterator<Item = Command> {
```

Note that `Option` implements `Iterator`, so single-commands can be wrapped in an option when returning them.